.github/workflows/deploy.yml

```diff
@@ -69,3 +76,23 @@ jobs:
           app-name: ${{ vars.AZURE_WEBAPP_NAME }}
           images: ghcr.io/${{ github.repository }}:${{ github.sha }}
           publish-profile: ${{ secrets.AZURE_WEBAPP_PUBLISH_PROFILE }}
+
+  release:
+    needs: deploy
+    if: ${{ github.ref_type == "tag" && !contains(github.ref, "-rc") }}
```

Comment:
I think these have to be single quotes :roll_eyes:

Reply:
Ah that's true, thanks for catching that!